BikeCSV/src/main/scala/by/itechart/Month.scala

```diff
@@ -1,16 +0,0 @@
-package by.itechart
-
-object Month extends Enumeration {
```

Comment:
Already defined in Java
https://docs.oracle.com/javase/8/docs/api/java/time/Month.html